src/main/java/TaskList.java

```diff
@@ -0,0 +1,42 @@
+public class TaskList {
+    private Task[] taskList = new Task[100];
+    private int taskCount = 0;
+
+    public void printLine(){
+        System.out.println("\t" + "-----------------------------------------");
+    }
+
+    public void addTask(Task task){
+        taskList[taskCount] = task;
+        taskCount += 1;
+    }
+
+    public void printTaskList(){
+        printLine();
+        System.out.println("\t" + " Here are the tasks in your list:");
+        for(int i = 0 ; i < taskCount; i++){
```

Comment:
I think there should be a space between ")" and "{", 
same thing throughout all the codes!